### PR TITLE
Audits view

### PR DESCRIPTION
**Pull request by jpwenzelc**

### CODE REVIEW

File: front-end/src/App.js [hunk 1]
```diff
@@ -1,190 +1,76 @@
...
-    if (!isUserLoggedIn()) {
+    if (!isUserLoggedIn) {
       // the api calls will fail because we're not authorized
       return;
     }
 
     api({ endpoint: `/config`, token: idToken })
       .then(response => response.json())
       .then(body =>
-        setLastMeetingThresholdsInWeeks(body.lastMeetingThresholdsInWeeks)
```
Comment (aclaudel) [inline front-end/src/App.js]: This shouldn't be removed

File: front-end/src/App.js [hunk 2]
```diff
@@ -1,190 +1,76 @@
...
-    return () => {
-      setSortAlgorithm(() => sortAlgorithmToUse);
-    };
-  }
+  const isUserLoggedIn = useCallback(() => {
+    return idToken;
+  }, [idToken]);
 
   const refreshConfig = useCallback(() => {
-    if (!isUserLoggedIn()) {
```
Comment (aclaudel) [inline front-end/src/App.js]: This shouldn't be removed

File: front-end/src/App.js (hunk 1 above)
Comment (edtheshed) [reply front-end/src/App.js]: fixed

File: front-end/src/App.js (hunk 2 above)
Comment (edtheshed) [reply front-end/src/App.js]: fixed, by removing

**Pull request merged**